sunpy/net/hek/hek.py

```diff
@@ -18,11 +23,15 @@
 from sunpy.time import parse_time
 from sunpy.util import dict_keys_same, unique
 from sunpy.util.xml import xml_to_dict
+from sunpy import __file__
 
 __all__ = ['HEKClient', 'HEKTable', 'HEKRow']
 
 DEFAULT_URL = 'https://www.lmsal.com/hek/her?'
+UNIT_FILE_PATH = Path(os.path.dirname(__file__)) / "net" / "hek"/ "unit_properties.json"
```

Comment:
Does this need the `/ "net" / "hek"/` ?

Reply:
@nabobalis changed this in the current version